apps/products-feed/src/modules/app-configuration/attribute-fetcher.ts

```diff
@@ -0,0 +1,49 @@
+import { Client } from "urql";
+import {
+  AttributeWithMappingFragmentFragment,
+  FetchAttributesWithMappingDocument,
+} from "../../../generated/graphql";
+
+export class AttributeFetcher {
```

Comment:
nice implementation, maybe we can make it generic and put to shared

Reply:
![image](https://github.com/saleor/apps/assets/5188636/624db056-16b9-4383-b6cf-b03321b96321)

Reply:
😆